pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/kit/KitStatusChangeRoute.java

```diff
@@ -34,12 +37,12 @@ public Object processRequest(Request request, Response response, String userId)
         DDPInstanceDto ddpInstanceDto = new DDPInstanceDao().getDDPInstanceByInstanceName(realm).orElseThrow();
         if (UserUtil.checkUserAccess(null, userId, "kit_shipping", userIdRequest) || UserUtil.checkUserAccess(null, userId, "kit_receiving",
                 userIdRequest)) {
-            scanResultList = new ArrayList<>();
+            List<ScanResult> scanResultList = new ArrayList<>();
```

Comment:
Instead of relying on instance variables, we create the payload and results objects inside of the method, thereby eliminating shared state across threads.